services/checks/checks.go

```diff
@@ -74,9 +74,11 @@ const (
 )
 
 // pmm-agent versions with known changes in Query Actions.
+// To match all pre-release versions add '-0' suffix to specified version.
 var (
-	pmmAgent260     = version.MustParse("2.6.0")
-	pmmAgent270     = version.MustParse("2.7.0")
+	pmmAgent2_6_0   = version.MustParse("2.6.0")
+	pmmAgent2_7_0   = version.MustParse("2.7.0")
+	pmmAgent2_27_0  = version.MustParse("2.27.0-0")
```

Comment:
According to https://semver.org/ numeric pre-pelease suffixes has the lowest priority, so by adding `-0` we match all pre-releases for the target version.

You can check order here: https://semvercompare.azurewebsites.net/?version=1.0.0-HEAD&version=1.0.0-alpha&version=1.0.0-beta&version=1.0.0-ALPHA&version=1.0.0-A&version=1.0.0&version=1.0.0-0